code/modules/surgery/bodyparts/species_parts/misc_bodyparts.dm

```diff
@@ -512,7 +512,7 @@
 	)
 	return ..()
 
-/obj/item/bodypart/arm/left/golem/set_owner(new_owner)
+/obj/item/bodypart/arm/left/golem/update_owner(new_owner)
```

Comment:
Why is this not `apply_ownership` and `clear_ownership`?

Reply:
I don't know tbh